website/docs/integrator-section/transferable-record/preparing-transferable-record.md

```diff
@@ -3,36 +3,37 @@ id: preparing-transferable-record
 title: Preparing Transferable Record
 sidebar_label: Preparing Transferable Record
 ---
+import WrapSingleDocument from "/src/reusables/_wrap-single-document.mdx";
 
 In this section, we will prepare the DNS and the content of the transferable record to be issued.
 
-## DNS Configuration
+## DNS configuration
 
 Similar to [binding the document store to a domain name](/docs/integrator-section/verifiable-document/ethereum/document-store), you will have to bind the identity of the token registry to a domain name.
 
-To do that simply create a `TXT` record on your domain with the following entry:
+To do that, create a `TXT` record on your domain with the following entry value:
 
-```txt
-openatts net=ethereum netId=3 addr=0x8431012Bc040942B59e3C5bf428221eab0b2f723
-```
+| Type | Domain      | Entry Value                                                     |
+| ---- | ----------- | --------------------------------------------------------------- |
+|`TXT` | example.com | "openatts net=`ethereum` netId=`11155111` addr=`0x8431012Bc040942B59e3C5bf428221eab0b2f723`" |
 
-You will need to replace the token registry address `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with the address you've got from the previous step.
+You will need to replace the token registry address `0x8431012Bc040942B59e3C5bf428221eab0b2f723` with the address you get from the previous step.
 
-If you like to use our sandbox DNS for the purpose of the exercise, you may instead run the following command, following the instructions from [earlier](/docs/integrator-section/verifiable-document/ethereum/dns-proof):
+If you want to use our sandbox DNS for the purpose of the exercise, run the following command instead following the instructions from [earlier](/docs/integrator-section/verifiable-document/ethereum/dns-proof):
```

Comment:
doesn't think break the user flow again as it will redirect to the ethereum section?

Reply:
Agree, updated to:
"If you want to use our sandbox DNS for the purpose of exercise, run the following command instead..."

Reply:
At the end of this page, added: 

> Additional reading
You will find more details about configuring DNS on [this page](http://localhost:3000/docs/integrator-section/verifiable-document/ethereum/dns-proof).